cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -0,0 +1,535 @@
+Feature: JWT Authenticator - Fetch signing key
+
+  Scenario: provider-uri is configured with valid value
```

Comment:
Its a copy of the test here not ?
  Scenario: provider-uri is configured with valid value

Reply:
Its not a copy, i just moved the original test because it belong to the new feature file i created